### PR TITLE
Ticket purchase branch

### PR DESCRIPTION
**Pull request by AngryMagnets**
Currently, it displays all the tickets matching an event, and on event creation, 2 tickets are created at a price of 29.99$ I have not yet added payment method validation or changing how many tickets a user wants to buy

### CODE REVIEW

File: PassIO/templates/checkout.html [hunk 1]
```diff
@@ -18,9 +18,9 @@
   <script src="../static/js/animate-in.js"></script>
   <script src="../static/js/smooth-scroll.js"></script>
 </head>
-<body>
-  {% include 'navbar.html' %}
```
Comment (Jared-Waldroff) [inline PassIO/templates/checkout.html:22]: Why was the navbar removed from the checkout page?

**Review by Jared-Waldroff — CHANGES_REQUESTED** [review]
Everything looks good, ticketing looks good, the changing of the meta data for html pages is good. The only thing I saw was the navbar was deleted from the checkout page. It should be left in as every page needs the navbar.

**Comment by BryceD-4** [comment]
Ahh now I see what you meant about hyperlinking the text as an <a> tag instead of making it a button. I am ok with this if everyone else is. I was planning to bring up changing the method of "events_display" to the "events.html" on Friday. Lets chat about the ticket purchasing feature as I don't want to end up working on something that you are already working on. Once Jared's comment above is resolved, I feel it is good to merge.

**Review by yukiiso — APPROVED** [review]

**Review by Jared-Waldroff — APPROVED** [review]
Good changes! I like the changes you made to be able to purchase specific tickets.

**Pull request merged**